Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -627,11 +608,10 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
 
             // Et calculation
             [=] AMREX_GPU_DEVICE (int i, int j, int /*k*/){
-#ifdef AMREX_USE_EB
                 // In RZ Et is associated with a mesh node, so we need to check if the mesh node is covered
                 amrex::ignore_unused(lt);
```

Comment:
We should remove this. `lt` is never used. We could remove it entirely this function. But even if we want to keep it just in case for future extension, we should move `amrex::ignore_unused` outside the device lambda so that it's not captured.

Reply:
good catch. will remove and add an inline comment above.